examples/configs/src/view-configs/3d-maps/blood-vessel-named.js

```diff
@@ -69,7 +69,7 @@ function generateBloodVesselConfig() {
   glomsFeatureSelectionScope.setValue(['Volume']);
 
 
-  config.linkViewsByObject([spatialThreeView, lcView, description], {
+  config.linkViewsByObject([spatialThreeView, lcView, linkController], {
     spatialTargetZ: 0,
```

Comment:
```suggestion
    spatialRenderingMode: '3D',
    spatialTargetZ: 0,
```

Did you want this configuration to initially be in 3D mode?

Reply:
No we need to start in 2D until the data is loaded, if we start 3D right away the segmentations are not showing up